packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -15,9 +23,24 @@ function getHeaders (config, application, reqType) {
   if (debug) {
     headers['dd-telemetry-debug-enabled'] = 'true'
   }
+  if (config.apiKey) {
+    headers['dd-api-key'] = config.apiKey
+  }
   return headers
 }
 
+function getAgentlessTelemetryEndpoint (site) {
+  if (site === 'datad0g.com') { // staging
+    return 'https://all-http-intake.logs.datad0g.com'
+  }
+  if (DATA_CENTERS.includes(site)) {
+    return `https://instrumentation-telemetry-intake.${site}`
+  }
+  if (site === 'datadoghq.eu') {
+    return 'https://instrumentation-telemetry-intake.eu1.datadoghq.com'
+  }
+}
+
```

Comment:
No default case ?
Also I feel like the `DATA_CENTERS` list might never get maintained properly. What if you simplified it like:

```suggestion
function getAgentlessTelemetryEndpoint (site) {
  if (site === 'datad0g.com') { // staging
    return 'https://all-http-intake.logs.datad0g.com'
  }
  if (site === 'datadoghq.eu') {
    return 'https://instrumentation-telemetry-intake.eu1.datadoghq.com'
  }
  
  return `https://instrumentation-telemetry-intake.${site}`
}

```

Reply:
I like it, thanks